src/report.py

```diff
@@ -1,97 +1,208 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: 2023 Fabian-Paul Utech <f.utech@gmx.net>
+# SPDX-FileCopyrightText: 2023 Ahmed Sheta <ahmed.sheta@fau.de>
 
-from reportlab.lib.pagesizes import A4
+import argparse
+import os
+
+import pandas as pd
 from reportlab.lib import colors
-from reportlab.platypus import SimpleDocTemplate, Table, TableStyle, Spacer, Paragraph
+from reportlab.lib.pagesizes import A4
 from reportlab.lib.styles import getSampleStyleSheet
-from reportlab.pdfbase import pdfmetrics
-from reportlab.lib.styles import ParagraphStyle
+from reportlab.platypus import Paragraph, SimpleDocTemplate, Spacer, Table, TableStyle
+
+standard_group_format = {
+    # 1 pdf per lead (1 row in .csv)
+    "Contact": ["Last Name", "First Name", "Company / Account", "Phone", "Email"],
+    "Reviews": [
+        "google_places_user_ratings_total",
+        "google_places_rating",
+        "google_places_price_level",
+        "reviews_sentiment_score",
+    ],
+    #'Region':[] starts with regional_atlas
+    # Regarding columns names if there are more than one '_' take the split after the second _
+}
+
+
+def process_lead(lead):
+    # Input search string (either specific leads or a whole file)
+    # Output: pd.series of a lead from leads_enriched.csv
+    try:
+        df = pd.read_csv("src/data/leads_enriched.csv", delimiter=",")
```

Comment:
Can this be extended to also allow S3 usage? Perhaps this can be combined with the new DAL to retrieve a certain lead regardless of storage location

Reply:
I will adjust it to the new documentation theme and S3 architecture in the next sprint. Then it should be also clear, how it works / how to test it

Reply:
It is a deprecated now. It is not compatible with the new version of the main.py . 
If needed can be executed simply by using python reports.py